.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Bug report
+about: Create a report to help us improve the library
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- Before raising, please check if somebody else has already reported your issue. -->
+
+### Describe the bug
+A clear and concise description of what the bug is.
+
+### Steps to reproduce
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+### Environment
+* Bugsnag version:
+* PHP version:
+* PSR version:
```

Comment:
This should probably be "PSR-3 version", or "`psr/log` version`" (referencing the composer package)

It might be easier to remove entirely — the version of the composer package doesn't matter because the interface hasn't changed so it can't have any bearing on the bug report. There's a new process for retroactively updating PSRs (previously they were set in stone), but nothing has changed for PSR-3 yet — we could add this line if/when that happens (we'd have to update the library to support multiple versions then, anyway)

We should also add "PSR-3 implementation and version" because it may be a bug with the library implementing PSR-3 causing issues, e.g.

> PSR-3 implementation and version: Monolog 1.2.3